### PR TITLE
Added SecWatch to the justice department.

### PR DESCRIPTION
**Pull request by Radezolid**
## About the PR
Added the SecWatch to the rest of the justice department, gone are the days of "hey why is this dude in jail?".
EDIT: This was requested in a [WYCI Suggestion](https://discord.com/channels/968983104247185448/1301883999345709126/1301883999345709126).
## Why / Balance
Justice needs to know why the person has been imprisoned. Prosecutor needs to know for better correction of sentencing, Clerk and CJ for warrants and notes.

## Technical details
Added the `SecWatchCartridge` to each of the justice department roles.

## Media

image

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

None.
**Changelog**

:cl:
- tweak: Now the prosecutor, clerk and chief justice PDAs comes with a SecWatch program installed.

### CODE REVIEW

**Comment by Proxysseia** [comment]
wahoo, yippie

**Review by deltanedas — APPROVED** [review]
they can use BaseSecurityPDA to inherit it

**Review by deltanedas — CHANGES_REQUESTED** [review]
a

**Comment by Radezolid** [comment]
> they can use BaseSecurityPDA to inherit it

They can but not everything security gets down the line will be needed by Justice, i do admit i could make a base PDA for justice and make them inherit from it.

File: Resources/Prototypes/Entities/Objects/Devices/pda.yml [hunk 1]
```diff
@@ -479,20 +479,14 @@
...
-  parent: BaseSecurityPDA
+  parent: BaseJusticePDA # DeltaV - Added secwatch to justice. Was BaseSecurityPDA.
   id: LawyerPDA
   name: lawyer PDA
   description: For lawyers to poach dubious clients.
   components:
   - type: Pda
     id: LawyerIDCard
     state: pda-lawyer
-    penSlot:
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Objects/Devices/pda.yml:490]: should have a comment saying penSlot was removed as its in parent

Comment (Radezolid) [reply Resources/Prototypes/Entities/Objects/Devices/pda.yml:490]: I forgot, will do.

**Review by deltanedas — APPROVED** [review]

**Review by deltanedas — APPROVED** [review]

**Pull request merged**